modules/authglobus/default-disable

```diff
@@ -0,0 +1,3 @@
+This file indicates that the default state of this module
```

Comment:
this should be enabled by default as it has to be specifically added.

Reply:
Wanted to follow the convention of the other 'optional' modules. No problem with changing this though.

Reply:
the ones built into the simplesamlphp are default-disabled, the ones that you add though an explicit require are default-enabled